tests/unit/config-performance.test.js

```
@@ -0,0 +1,505 @@
+
+			const endTime = performance.now();
+			const duration = endTime - startTime;
+
+			// Should complete within reasonable time even under stress
+			expect(duration).toBeLessThan(1000); // 1 second for 500 mixed operations
+			expect(operations).toHaveLength(500);
+		});
+
+		test('should handle extremely deep nesting without stack overflow', () => {
+			// Create config with very deep nesting
+			let deepConfig = {};
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'deepConfig' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)